modules/CmpdReg/src/client/src/MetaLot.js

```diff
@@ -431,7 +431,12 @@ $(function() {
 	    },
 
 	    newLot: function () {
-		    window.open("#register/" + this.lotController.model.get('corpName'));
```

Comment:
It's a little risky to split on "-" since the separators between prefix, parent number, and lot number are all configurable. Can you get the `parent` from somewhere within the `lotController.model` so you can directly pull the parent corpName rather than doing string manipulation on the lot corpName?

Reply:
Tried to add your feedback to each of the affected areas. Had to do a little differently in each place.